dev/tasks/r/github.nightly.yml

```diff
@@ -0,0 +1,381 @@
+# Licensed to the Apache Software Foundation (ASF) under one

```

Comment:
This file should not be called "nightly" because it can also be run on a PR. maybe something about "packages"?

Reply:
With regards to my other comment, I think the name is fine.

Reply:
To expand on this: I think it is fine because being run for the nightlies is the main purpose of this workflow, it can now be run on PRs but I still think it would make more sense to extend the functionality of `r.yml` instead.